modin/experimental/core/execution/native/implementations/hdk_on_native/db_worker.py

```diff
@@ -12,11 +12,17 @@
 # governing permissions and limitations under the License.
 
 """Module chooses a proper worker class."""
+import warnings
 
 try:
     # Try to use PyHDK by default
-    from .pyhdk_worker import PyHdkWorker as DbWorker
+    from .hdk_worker import HdkWorker as DbWorker
 except ModuleNotFoundError:  # fallback to pyomniscidbe package
     from .omnisci_worker import OmnisciWorker as DbWorker
 
+    warnings.warn(
```

Comment:
Let's just say "is deprecated" since in fact we will not support omnisci backend in upcoming release. and let's mention explicitly how to specify hdk backend, smth like "use MODIN_STORAGE_FORMAT="hdk" now"